packages/ui/cypress/support/index.ts

```diff
@@ -0,0 +1,18 @@
+import faker from '@faker-js/faker'
+// Needed to process uno styles
+import 'uno.css'
+import 'd3-graph-controller/default.css'
+import 'splitpanes/dist/splitpanes.css'
+import '@unocss/reset/tailwind.css'
+import 'codemirror/lib/codemirror.css'
+import 'codemirror-theme-vars/base.css'
+import 'tippy.js/dist/tippy.css'
+import '../../client/styles/main.css'
```

Comment:
Could we share them somewhere?

Reply:
You can but you'd need to create like... another entry file. I can do it, I just never really need to in other projects. There aren't usually a lot of global deps.